File: Praktikum Pemrograman 1/Praktikum pemrograman 1 modul 4/Python/PRAK404-2410817110006-HamkaArifani.py
print("Pilih program\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit\n")
pilihan=int(input("Masukkan pilihan: "))
while True:
    if pilihan==1:
        A=float(input("Masukkan nilai pertama: "))
        B=float(input("Masukkan nilai kedua: "))
        At=A+B
        print(f"Hasil penjumlahan antara {A:.2f} dengan {B:.2f} adalah {At:.2f}")
        break
    elif pilihan==2:
        A=float(input("Masukkan nilai pertama: "))
        B=float(input("Masukkan nilai kedua: "))
        Ak=A-B
        print(f"Hasil penjumlahan antara {A:.2f} dengan {B:.2f} adalah {Ak:.2f}")
        break
    elif pilihan==3:
        A=float(input("Masukkan nilai pertama: "))
        B=float(input("Masukkan nilai kedua: "))
        Ap=A*B
        print(f"Hasil penjumlahan antara {A:.2f} dengan {B:.2f} adalah {Ap:.2f}")
        break
    elif pilihan==4:
        A=float(input("Masukkan nilai pertama: "))
        B=float(input("Masukkan nilai kedua: "))
        Ab=A/B
        print(f"Hasil penjumlahan antara {A:.2f} dengan {B:.2f} adalah {Ab:.2f}")
        break
    elif pilihan==5:
        print("Terimakasih, telah menggunakan kalkulator HAMKAARIFANI")
        break
    else:
        print("Input anda salah, silahkan coba lagi")
        break

File: Praktikum Pemrograman 1/Praktikum pemrograman 1 modul 4/C/PRAK404-2410817110006-HamkaArifani.c
#include <stdio.h>
int main(){
    int pilihan;
    float  A, B, At, Ak, Ap, Ab;
    printf("Pilih program\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit\n");
    printf("Masukkan pilihan: ");
    scanf("%d", &pilihan);
    while(1){
        if(pilihan==1){
            printf("Masukkan nilai pertama: ");
            scanf("%f", &A);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &B);
            At= A+B;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f", A, B, At);
            break;
        }
        else if(pilihan==2){
            printf("Masukkan nilai pertama: ");
            scanf("%f", &A);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &B);
            Ak= A-B;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f", A, B, Ak);
            break;
        }
        else if(pilihan==3){
            printf("Masukkan nilai pertama: ");
            scanf("%f", &A);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &B);
            Ap= A*B;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f", A, B, Ap);
            break;
        }
        else if(pilihan==4){
            printf("Masukkan nilai pertama: ");
            scanf("%f", &A);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &B);
            Ab= A/B;
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f", A, B, Ab);
            break;
        }
        else if(pilihan==5){
            printf("Terimakasih, telah menggunakan kalkulator HAMKAARIFANI");
            break;
        }
        else{
            printf("Input anda salah, silahkan coba lagi");
            break;
        }
    }
}